test/Sets/Polygon.jl

```diff
@@ -190,6 +190,14 @@ for N in [Float64, Float32, Rational{Int}]
                                     [N[2, -2], N[6, 0], N[8, 2],
                                      N[8, 4], N[6, 6], N[2, 8]])
 
+        P = VPolygon([N[0, -1], N[2, 0], N[1, 1], N[0, 1]], apply_convex_hull=false)
+        Q = VPolygon([N[0, 1], N[0, -1]], apply_convex_hull=false)
+        PQ = minkowski_sum(P, Q)
+        @test length(PQ.vertices) == 5
+        @test is_cyclic_permutation(PQ.vertices,
+                                    [N[0, -2], N[2, -1], N[2, 1],
+                                     N[1, 2], N[0, 2]])
```

Comment:
This test actually passes in `master`. Was something forgotten?

Reply:
Hmm. On master `_σ_helper(EAST, vlistQ)` could return either of the vertices of `Q`, so it might just be it returns the right one? How about the order of vertices in `Q` swapped?

Reply:
I made a mistake in the test that the facet needs to be to the east. (a test with a facet to the west probably wouldn't hurt.).

Reply:
I confirmed that the test cases fail on master but pass with this fix.